arch/sim/Kconfig

```diff
@@ -265,6 +265,29 @@ endchoice
 
 endif
 
+config SIM_VIDEO
```

Comment:
@PeterBee97 let's split sim related change to new PR

Reply:
OK (well let me pass this CI first..)